app/controllers/concerns/course/forum/topics_hide_concern.rb

```diff
@@ -0,0 +1,25 @@
+module Course::Forum::TopicsHideConcern
```

Comment:
@lowjoel Actually what's the the name convention of concern? I prefer `TopicsHideConcern` over `TopicControllerHideConcern`,`"Controller"` is redundant since all of them are concerns of controllers.